packages/grafana-ui/src/components/TimePicker/TimeRangePicker/TimeRangeForm.tsx

```diff
@@ -24,7 +24,7 @@ interface Props {
   isReversed?: boolean;
 }
 
-interface InputState {
+export interface InputState {
```

Comment:
Does this a doc comment?

Reply:
No, it is not exported from the package so it should be fine.